plugin/powerControl/powerControl.yaml

```
@@ -0,0 +1,43 @@
+
+PowerControl:
+  PredictorScript: YOUR_PREDICTOR_SCRIPT
+  EnableSleep: true
+  SleepTimeThresholdSeconds: 70     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)